### ISSUE TITLE
Installing Pgquarrel in Windows - VS 2017 Error

### ISSUE THREAD

**Issue by burceviks** [eulerto/pgquarrel#36]
Hi

I have cloned the project from github, and tried to run cmake.  In my machine I have Visual Studio 2017 Professional Edition is installed as well as PostGresSql 10.2 version is installed.

I have clicked on Configuration button on CMake after selecting the source directory. It said as screenshot says below. 

I am not quite sure what to do next, would you be able to help ?
Thanks

![image](https://user-images.githubusercontent.com/4746283/36329199-f3682630-1319-11e8-8814-714451bd9b96.png)

**Comment by eulerto** [comment]
@burceviks Could you share the contents of the file show at the screenshot footer?

**Comment by eulerto** [comment]
Looking at your screenshot again it seems you forgot to **Add Entry** CMAKE_PREFIX_PATH (if your postgres installation is not in your PATH) and also CMAKE_INSTALL_PREFIX.

**Comment by burceviks** [comment]
Hi

Error at the buttom of the screen is at the buttom of the screen.

I had added the path to my environment variables.
[image: Satır içi resim 1]

Also I had tried to add native compiler as below too, but didn't make any
change.
[image: Satır içi resim 2]

I have msbuild.exe in that directory.

Thanks


Selecting Windows SDK version to target Windows 10.0.10586.

CMake Error at CMakeLists.txt:3 (project):
Failed to run MSBuild command:

MSBuild.exe

to get the value of VCTargetsPath:

The system cannot find the file specified


Configuring incomplete, errors occurred!

See also "C:/Users/burcu/work/pgquarrel/pgquarrel/src/CMakeFiles/CMakeFiles/
CMakeOutput.log".


2018-02-16 14:28 GMT-08:00 Euler Taveira de Oliveira <
notifications@github.com>:
